tests/kitchenstories.py

```diff
@@ -0,0 +1,77 @@
+from recipe_scrapers.kitchenstories import KitchenStories
+from tests import ScraperTest
+
+
+class TestKitchenStoriesScraper(ScraperTest):
```

Comment:
A request to add test coverage on the `language` method given the multilingual site content.

Reply:
Should I put multiple html files and add multiple tests for each language?